### DISCUSSION TITLE
How modify messages from auth ?

### DISCUSSION THREAD

**Discussion by Reptiluka** [adonisjs/core#1840] [Help]
Hi !
I use the code from the docs for login : 

```
public async store ({auth, request, response} : HttpContextContract) {
        const email = request.input('email')
        const password = request.input('password')
        await auth.attempt(email, password)

        response.redirect('/')
    }
```

How I modify the errors message from validation ??

**Comment by hariprasath-yadav** [comment]
Use **Exception Handlers** in V5 to use your own error messages
Refer to this page if you use V5
https://preview.adonisjs.com/guides/http/exception-handling
